Frontend/voitheia-App/src/variables.scss

```diff
@@ -6,38 +6,36 @@ $gray-3: #848484;
 $gray-4: #585858;
 $gray-5: #151515;
 
-$green: #6FB371;
-$red: #d90000;
-$dark-red: #850000;
-$dark-red-2: #a94442;
-$light-red-1: #ffe3e3;
-$light-red-2: #fcb8b8;
-$light-red-3: #f2dede;
-$blue: #00a0de;
-$white: #FFFFFF;
-
 //Brand Color
-$brand-color: #78E9FF;
-$brand-color-light: #78E9FF;
-$brand-color-dark: #4BA2B3;
+$brand-color: #20cde4; //#FF9985;
+$brand-color-light: #78E9FF; //#FF9985;
+$brand-color-dark: #078fa1;//#FF9985;
 
-$brand-secondary-color: #91FF96;
-$brand-secondary-color-light: #91FF96;
-$brand-secondary-color-dark: #6FB371;
+$brand-secondary-color: #f4623a;
+$brand-secondary-color-light: #f57d5b;
+$brand-secondary-color-dark: #e64215;
 $brand-secondary-color-dark-dark:$black;
 
 $brand-warn-color:#ff0000;
 $brand-warn-color-light:#ffb3b3;
 $brand-warn-color-dark:$brand-warn-color;
 
-$brand-background-color: #FF9985;
-
```

Comment:
Brand colors should be as shown in our related issue #18 .
Brand color: #78E9F
Brand color dark: #4BA2B3
Secondary: #91FF96
Secondary dark: #6fB371

Reply:
brand colors corrected